x-pack/plugins/ingest_manager/common/constants/epm.ts

```diff
@@ -7,3 +7,4 @@
 export const PACKAGES_SAVED_OBJECT_TYPE = 'epm-packages';
 export const INDEX_PATTERN_SAVED_OBJECT_TYPE = 'index-pattern';
 export const INDEX_PATTERN_PLACEHOLDER_SUFFIX = '-index_pattern_placeholder';
+export const MAX_TIME_COMPLETE_INSTALL = 30000;
```

Comment:
30 seconds seems short considering the network is involved, but it's easy enough to change and you mentioned the steps you took to test.

Reply:
I'm open, wasn't  too sure about this.  Perhaps 1 minute?  That could give ES a change to return a timeout if there is an issue there and we can handle that separately.

Reply:
accidentally edited your comment when editing mine somehow.  perhaps 1 minute would be sufficient and give some time for potential timeout errors.